packages/commonwealth/client/scripts/views/components/component_kit/cw_icons/cw_icons.tsx

```diff
@@ -1233,6 +1293,36 @@ export const CWDots = (props: IconProps) => {
   );
 };
 
+export const CWDotsHorizontal = (props: IconProps) => {
+  const {
+    className,
+    componentType,
+    disabled,
+    iconButtonTheme,
+    iconSize,
+    selected,
+    ...otherProps
+  } = props;
+  return (
+    <svg
+      className={getClasses<IconStyleProps>(
+        { className, disabled, iconButtonTheme, iconSize, selected },
+        componentType
+      )}
+      xmlns="http://www.w3.org/2000/svg"
+      width="16"
+      height="16"
+      fill="none"
+      viewBox="0 0 16 16"
+      {...otherProps}
+    >
+      <path d="M8.96875 8C8.96875 8.53503 8.53503 8.96875 8 8.96875C7.46497 8.96875 7.03125 8.53503 7.03125 8C7.03125 7.46497 7.46497 7.03125 8 7.03125C8.53503 7.03125 8.96875 7.46497 8.96875 8Z" fill="#656167" stroke="#656167" stroke-width="0.0625"></path>
+      <path xmlns="http://www.w3.org/2000/svg" d="M4 9C4.55228 9 5 8.55228 5 8C5 7.44772 4.55228 7 4 7C3.44772 7 3 7.44772 3 8C3 8.55228 3.44772 9 4 9Z" fill="#656167"></path>
+      <path xmlns="http://www.w3.org/2000/svg" d="M12 9C12.5523 9 13 8.55228 13 8C13 7.44772 12.5523 7 12 7C11.4477 7 11 7.44772 11 8C11 8.55228 11.4477 9 12 9Z" fill="#656167"></path>
+    </svg>
+  );
+};
+
```

Comment:
1. prettier: file is not formatted correctly
2. Please do not use properties with dashes. They should all be transformed to camelCase properties. Otherwise, we have a lot of warnings and noise in the console. So `stroke-width` => `strokeWidth` etc